public/app/features/annotations/annotations_srv.ts

```diff
@@ -6,10 +6,21 @@ import './editor_ctrl';
 import coreModule from 'app/core/core_module';
 // Utils & Services
 import { dedupAnnotations } from './events_processing';
+import _ from 'lodash';
+
 // Types
 import { DashboardModel } from '../dashboard/state/DashboardModel';
-import { AnnotationEvent, AppEvents, DataSourceApi, PanelEvents, PanelModel, TimeRange } from '@grafana/data';
+import {
+  AnnotationEvent,
+  AppEvents,
+  DataSourceApi,
+  PanelEvents,
+  PanelModel,
+  ScopedVars,
+  TimeRange,
+} from '@grafana/data';
 import { getBackendSrv, getDataSourceSrv } from '@grafana/runtime';
+import { TemplateSrv } from '../templating/template_srv';
```

Comment:
You should just use the singleton instead of injecting it.
```suggestion
import templateSrv from '../templating/template_srv';
```